sources/packages/backend/apps/api/src/route-controllers/education-program-offering/_tests_/e2e/education-program-offering.institutions.controller.validateOffering.e2e-spec.ts

```diff
@@ -111,6 +117,121 @@ describe("EducationProgramOfferingInstitutionsController(e2e)-validateOffering",
+        hasOfferingWILComponent: "no",
+        studyStartDate: "2023-09-01",
+        studyEndDate: "2024-06-30",
+        lacksStudyBreaks: false,
+        studyBreaks: [
+          {
+            breakStartDate: "2023-12-01",
+            breakEndDate: "2023-12-18",
+          },
+        ],
+        offeringType: OfferingTypes.Public,
+        offeringDeclaration: true,
+        actualTuitionCosts: 100001,
+        programRelatedCosts: 3211,
+        mandatoryFees: 456,
+        exceptionalExpenses: 555,
+      };
+
+      // Act/Assert
+      await request(app.getHttpServer())
+        .post(endpoint)
+        .send(payload)
+        .auth(institutionUserToken, BEARER_AUTH_TYPE)
+        .expect(HttpStatus.OK)
+        .expect({
+          offeringStatus: OfferingStatus.CreationPending,
+          errors: [],
+          infos: [],
+          warnings: [
+            {
+              typeCode: OfferingValidationWarnings.OfferingCostExceedMaximum,
+              message: "actualTuitionCosts must not be greater than 100000",
+            },
+          ],
+          studyPeriodBreakdown: {
+            fundedStudyPeriodDays: 304,
+            totalDays: 304,
+            totalFundedWeeks: 44,
+            unfundedStudyPeriodDays: 0,
+          },
+        });
+    },
+  );
+
+  it(`Should return error and warning when data with programRelatedCosts greater than ${MONEY_VALUE_FOR_UNKNOWN_MAX_VALUE} is passed.`, async () => {
+    // Arrange
+    const institutionUserToken = await getInstitutionToken(
+      InstitutionTokenTypes.CollegeFUser,
+    );
+
+    const endpoint = `/institutions/education-program-offering/location/${collegeFLocation.id}/education-program/${collegeFEducationProgram.id}/validation`;
+    const payload = {
+      offeringName: "Offering validation test 2",
+      yearOfStudy: 1,
+      showYearOfStudy: true,
+      offeringIntensity: OfferingIntensity.fullTime,
+      offeringDelivered: OfferingDeliveryOptions.Online,
+      hasOfferingWILComponent: "no",
+      studyStartDate: "2023-09-01",
+      studyEndDate: "2024-06-30",
+      lacksStudyBreaks: false,
+      studyBreaks: [
+        {
+          breakStartDate: "2023-12-01",
+          breakEndDate: "2023-12-18",
+        },
+      ],
+      offeringType: OfferingTypes.Public,
+      offeringDeclaration: true,
+      actualTuitionCosts: 5000,
+      programRelatedCosts: 1000000,
+      mandatoryFees: 456,
+      exceptionalExpenses: 555,
+    };
+
+    // Act/Assert
+    await request(app.getHttpServer())
+      .post(endpoint)
+      .send(payload)
+      .auth(institutionUserToken, BEARER_AUTH_TYPE)
+      .expect(HttpStatus.OK)
+      .expect({
+        errors: ["Program related costs must be not greater than 999999."],
+        infos: [],
+        warnings: [
+          {
+            typeCode: OfferingValidationWarnings.OfferingCostExceedMaximum,
+            message: "programRelatedCosts must not be greater than 100000",
```

Comment:
That's a bit weird because if the `programRelatedCosts` must not be greater than 100000, they obviously cannot be grater than 999999. Should we show both messages in this case?

Reply:
As mentioned in the description, its the cost is greater than 100000, Institution can still create an offering with `creation pending` status. If the cost is greater than 999999, Api error will happen, so, they can't create an offering with cost greater than 999999 (this 2nd validation error was already there, it is for database safety purpose)

Reply:
ok. Thanks for the explanation.